spec/index.bs

```diff
@@ -2089,7 +2096,21 @@ enum GPUTextureAspect {
 
         **Returns:** |view|, of type {{GPUTextureView}}.
 
-        Issue: write definition. |this| |descriptor| |view|
+        1. If any of the following conditions are unsatisfied, generate a validation
+            error and return an [=invalid=] {{GPUTextureView}}.
+            <div class=validusage>
+                - 0 <= |baseMipLevel| < {{GPUTextureDescriptor/mipLevelCount}}
```

Comment:
we should probably use the special symbols instead of "<=" and "<" to avoid confusing the parsers